### PR TITLE
feat: add `dragon-editor` module

### LINKED ISSUE #1055: [Module Listing Request]: `dragon-editor`
### Description

`dragon-eidotr` is WYSIWYG editor.

### Repository

https://github.com/lovefields/dragonEditor

### npm

https://www.npmjs.com/package/dragon-editor

### Nuxt Compatibility

Nuxt 3

### PR DESCRIPTION
**Pull request by lovefields**
### 🔗 Linked issue

resolves #1055 

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

I just made the DragonEditor because I needed the WYSIWYG Editor to write on my [blog](https://dico.me/).

[Document](https://lovefields.github.io/dragonEditor-doc)
[Demo](https://lovefields.github.io/dragonEditor-doc/demo)

### CODE REVIEW

**Comment by atinux** [comment]
It would be nice to have a playground to see how the editor looks like.

Thank you for contributing to Nuxt :)

**Comment by lovefields** [comment]
@atinux Oh, Thanks.
I added [Demo page](https://lovefields.github.io/dragonEditor-doc/demo) in document

**Comment by lovefields** [comment]
@atinux Is someone checking this PR?

File: modules/dragon-editor.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Dragon Editor
```
Comment (atinux) [inline modules/dragon-editor.yml:1]:
```suggestion
name: dragon-editor
```

File: modules/dragon-editor.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+description: Javascript WYSIWYG editor in Nuxt3!
```
Comment (atinux) [inline modules/dragon-editor.yml:2]:
```suggestion
description: Javascript WYSIWYG editor in Nuxt!
```

File: modules/dragon-editor.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: dragon-editor
+description: Javascript WYSIWYG editor in Nuxt!
+repo: lovefields/dragonEditor
+npm: dragon-editor
+icon: ''
+github: https://github.com/lovefields/dragonEditor
+website: https://github.com/lovefields/dragonEditor
+learn_more: ''
+category: UI
```
Comment (atinux) [inline modules/dragon-editor.yml:9]:
```suggestion
category: Libraries
```

**Pull request merged**